iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergFilterFactory.java

```diff
@@ -190,6 +190,7 @@ private static int daysFromTimestamp(Timestamp timestamp) {
   // We have to use the LocalDateTime to get the micros. See the comment above.
   private static long microsFromTimestamp(Timestamp timestamp) {
     // `org.apache.hadoop.hive.common.type.Timestamp.valueOf(lit.toString()).toSqlTimestamp()`
+    // since HIVE-21862 changes literal parsing to UTC based timestamps
```

Comment:
The comment in this form does not make sense to me 😢 
Either:
```
    // HIVE-21862 changes literal parsing to UTC based timestamps to this:
    // `org.apache.hadoop.hive.common.type.Timestamp.valueOf(lit.toString()).toSqlTimestamp()`
```
Or just remove the comment?

Reply:
You're right, removed it